### DISCUSSION TITLE
CPU usage of strongswan non-visible in htop

### DISCUSSION THREAD

**Discussion by alex-airspan** [strongswan/strongswan#1411] [Q&A]
Hi

I run server/client on ubuntu and push 1G E2E TCP traffic. Traffic forwarded without issues, but cpu usage looks that in idle. 
in htop I enable show kernel process. but nothing show. 

Below printscreen of filter on charon process 

![image](https://user-images.githubusercontent.com/90091400/201344837-a437015f-fb7d-47b8-8403-57ca201852e8.png)

How possible see the CPU usage? 

Thanks.

**Comment by Thermi** [comment]
That's because processing is done by the kernel. Look at the kernel threads.

**Comment by alex-airspan** [comment]
Can you please share command how see kernel threads ?
I try  "top -H -b -n 1" and htop - nothing show it.

**Comment by Thermi** [comment]
Press Shift+K in htop.
